### PR TITLE
fix: increase width on validation labels

### PR DESCRIPTION
**Pull request by sjohnson-sage**
### Proposed behaviour

When the labelsInline prop is true on the DateRange component the validation icons should be displayed on the same line as the label text like this: 
image

### Current behaviour

The validation icons are being displayed on a second line underneath the label: 
image
image

### Checklist

- [x] Screenshots are included in the PR
- [x] Carbon implementation and Design System documentation are congruent
- [x] All themes are supported
- [x] Commits follow our style guide
- <del>[ ] Unit tests added or updated</del>
- <del>[ ] Cypress automation tests added or updated</del>
- <del>[ ] Storybook added or updated</del>
- <del>[ ] Typescript `d.ts` file added or updated</del>

### CODE REVIEW

File: src/__experimental__/components/date-range/date-range.component.js [hunk 1]
```diff
@@ -170,12 +172,14 @@ class DateRange extends React.Component {
           onFocus={ this.focusStart }
           data-element='start-date'
           ref={ this.startDateInputRef }
+          labelWidth={ this.inlineLabelWidth } // only applied when labelsInLine prop is true
```
Comment (jamime) [inline src/__experimental__/components/date-range/date-range.component.js:175]:
```suggestion
          labelWidth={ this.inlineLabelWidth } // Textbox only applies this when labelsInLine prop is true
```

File: src/__experimental__/components/date-range/date-range.component.js [hunk 2]
```diff
@@ -170,12 +172,14 @@ class DateRange extends React.Component {
...
         />
         <DateInput
           { ...this.dateProps('end') }
           onFocus={ this.focusEnd }
           data-element='end-date'
           ref={ this.endDateInputRef }
+          labelWidth={ this.inlineLabelWidth } // only applied when labelsInLine prop is true
```
Comment (jamime) [inline src/__experimental__/components/date-range/date-range.component.js:182]:
```suggestion
          labelWidth={ this.inlineLabelWidth } // Textbox only applies this when labelsInLine prop is true
```

**Comment by harpalsingh** [comment]
@sjohnson-sage Can't seem to see the demo of this in chromatic?

Screenshot 2020-06-25 at 09 49 20

**Comment by sjohnson-sage** [comment]
@harpalsingh that's strange - I can see it!

**Comment by jamime** [comment]
@harpalsingh I can also see it.

image

**Pull request merged**